feat: added support for Gemini 1.5 Flash

roman-romanov-o opened the pull request:
Closes https://github.com/epam/ai-dial-adapter-vertexai/issues/87

Commit:
Add Gemini 1.5 flash model

roman-romanov-o commented on aidial_adapter_vertexai/deployments.py:
```diff
@@ -14,6 +14,7 @@ class ChatCompletionDeployment(str, Enum):
     GEMINI_PRO_1 = "gemini-pro"
     GEMINI_PRO_VISION_1 = "gemini-pro-vision"
     GEMINI_PRO_1_5 = "gemini-1.5-pro-preview-0409"
+    GEMINI_FLASH_1_5 = "gemini-1.5-flash"
```
Not sure, what version to add here for prod

In docs there is three options:

- Latest version: `gemini-1.5-flash-latest`
- Latest stable version: `gemini-1.5-flash`
- Stable version: `gemini-1.5-flash-001`

adubovik replied:
Let's use the stable version `gemini-1.5-flash-001`.

roman-romanov-o replied:
Done

Commit:
add flash model to integration tests

roman-romanov-o commented:
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/483790)

roman-romanov-o commented:
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/483836)

adubovik commented:
Please update README

adubovik commented on aidial_adapter_vertexai/chat/gemini/adapter.py:
```diff
@@ -111,6 +114,10 @@ async def parse_prompt(
                 return await Gemini_1_5_Pro_Prompt.parse(
                     self.file_storage, tools, messages
                 )
+            case ChatCompletionDeployment.GEMINI_FLASH_1_5:
```
No need to complicate the code by introduction of the alias class `Gemini_1_5_Flash_Prompt`.

```python
            case (
                ChatCompletionDeployment.GEMINI_PRO_1_5
                | ChatCompletionDeployment.GEMINI_1_5_FLASH
            ):
                return await Gemini_1_5_Pro_Prompt.parse(
                    self.file_storage, tools, messages
                )
```

roman-romanov-o replied:
Ok, maybe then let's rename this prompt parser to something like `Gemini_1_5_Prompt`?

roman-romanov-o replied:
Fixed, renamed `Gemini_1_5_Pro_Prompt` to `Gemini_1_5_Prompt`

Commit:
update README

Commit:
- change version of flash model to stable
- remove 1.5 flash prompt alias class
- Rename 1.5 pro prompt into just 1.5 prompt

Commit:
fix readme mistake

roman-romanov-o commented:
> Please update README

done

roman-romanov-o commented:
/deploy-review
>Environment URL: https://chat-ai-dial-adapter-vertexai-pr-88.nightly-test.deltixhub.io
>E2E tests status: success
